CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.14.0-dev
 
+### Fixed
+
+- Creating new tabs using `Cmd + T` on macOS is only enabled for decorations
+  set to full
```

Comment:
```suggestion
- Crash when trying to create a new tab without decorations enabled
```

Should actually document what we fixed, not how we fixed it.

Reply:
My bad, a previous commit had this but I forgot it when I undid the formatting i accidentally committed.